rector.php

```diff
@@ -25,5 +26,6 @@
     $rectorConfig->skip([
         ClosureToArrowFunctionRector::class,
         ReadOnlyPropertyRector::class,
+        SensitiveHereNowDocRector::class,
```

Comment:
Why? Seems, it's useful rule.

Reply:
It's OK, because "HTML" is used in content. Rename wrapper is safely.

Reply:
The indentation also changes, and in my opinion, the file becomes disorganized. I think that the way it is now is in line with the style we use.

Reply:
done.